docs/orchestrating-elastic-stack-applications/elasticsearch/reserved-settings.asciidoc

```diff
@@ -22,7 +22,6 @@ The following Elasticsearch settings are managed by ECK:
 * `xpack.security.authc.reserved_realm.enabled`
 * `xpack.security.enabled`
 * `xpack.security.http.ssl.certificate`
-* `xpack.security.http.ssl.client_authentication`
```

Comment:
I wonder if it's best to leave this our from the list of ECK managed settings, or if we should explain what values are supported / not supported.

Reply:
+1 to explain what values are supported / not supported. It doesn't seem to be a big effort to maintain and that shouldn't be something that changes a lot.

Not sure about the wording:
* `xpack.security.http.ssl.client_authentication` (supported for `none` or `optional` but not `required`)

Reply:
I would probably add new section "Partially restricted settings" and say `xpack.security.http.ssl.client_authentication` (supported values `none`, `optional`)